src/stm32l0/stm32l0-standard.ld

```diff
@@ -0,0 +1,22 @@
+/*
+ * 2020 - Karl Palsson <karlp@tweak.net.au>
+ * Considered to be released into the public domain, or where not available
+ * under your choice of the following spdx identifiers:
+ * MIT, ISC, Apache-2.0, BSD-1-Clause, BSD-2-Clause, BSD-3-Clause,
+ * CC-BY-4.0, GPL-2.0-or-later, LGPL-2.0-or-later
+ * Pick whatever makes your integration life easier
+ *
+ * Linker script for STM32L0 parts. Bootloader only for FIXME8k, and all parts
+ * have at least 8k RAM, (Of the L0's that have USB!)
+ */
+
+/* Define memory regions. */
+MEMORY
+{
+	vectors (rx) : ORIGIN = 0x08000000, LENGTH = 0x150
+	rom (rx)  : ORIGIN = 0x08000150, LENGTH = 0x1EB0
+	ram (rwx) : ORIGIN = 0x20000000, LENGTH = 8K
+}
+
+/* Include the common ld script. */
+INCLUDE stm32l1/stm32l1-base.ld
```

Comment:
this is wrong, but harmless, as the "base" file is actually the same "cortex-m-generic" file anyway.